packages/office-ui-fabric-react/src/components/Persona/PersonaCoin.tsx

```diff
@@ -143,6 +134,44 @@ export class PersonaCoin extends React.Component<IPersonaProps, IPersonaState> {
     );
   }
 
+  @autobind
+  private _onRenderCoin(props: IPersonaProps, size: number): JSX.Element {
+    if (props.onRenderCoin) {
+      console.log(size);
```

Comment:
Probably wasn't meant to be included in the commit.

Reply:
Yes, I'll clean that up. I was attempting to make a slightly better example to show. If you have any ideas on that let me know.